test/unit/jest.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
 	transform: {
 		'^.+\\.[jt]sx?$': '<rootDir>/test/unit/scripts/babel-transformer.js',
 	},
+	transformIgnorePatterns: [ 'node_modules/(?!(is-plain-obj))' ],
```

Comment:
Interesting, is Jest still having issues with ESM modules?

Reply:
Yes! If we ditch that, a bunch of unit tests where this module is directly or transitively used will fail.